src/app/tab/date/date.component.html

```diff
@@ -1,6 +1,49 @@
-<div class="dateContent"
+<div
+  class="dateContent"
   [style.fontSize]="shared.getFontSize(settings.config.date.scaling, 3)">
-  {{getDate()}}
+  <div *ngIf="!settings.config.date.short.enabled">
+    {{ getDate() }}
+  </div>
+
+  <div  *ngIf="settings.config.date.short.enabled" 
+        [ngSwitch]="settings.config.date.format">
+    {{ getWeekday() }}
+    <span *ngSwitchCase="10"><!-- dd/MM/YYYY -->
+      {{ getDay() }}
+      <span [ngClass]="{ dim: settings.config.date.dimDelimiter }">
+        {{ getDelimiter() }}
```

Comment:
This doesn't take into account if the day is not included but the month and year are, then there should only be a single delimiter shown, same if the day and month are both disabled, there should be no delimiter shown. I know it sounds odd, but the options are there for users to do that :)